### ISSUE TITLE
macos 12.6.6 install failed 

### ISSUE THREAD

**Issue by mani2care** [utmapp/UTM#5398]
macos 12.3.2 i have the ipsw file and installed successfully but i tried update the macos to 12.6.6 its failed after the restart nothing is showing the screen its block. i tried 3 more times still the same issue 

and i downloaded the full installer 12.6.6 and tried install still the same issue not updating to latest version its struck after the restart and unable to restore also.

**Comment by kMikaZu** [comment]
Also see [this](https://github.com/utmapp/UTM/issues/5307) issue. Apparently, this is a widespread problem...

**Comment by mani2care** [comment]
yes looks like same i tried fresh image with 12.6.6 and failed after the upgrade its block screen.

**Comment by sonofmom** [comment]
Same here, black screen after attempt to update OSX from 12.6.2 to 12.6.7, this is serious problem, I suggest that everyone who is running OSX 12.x guests should avoid updates for now as those break VMs. Can't speak about 13.x

My host is Ventura 13.4.1 with UTM version 4.2.5

**Comment by kMikaZu** [comment]
Seems like the Virtual Machines Hub app suffers from the same problem. Must be an Apple problem or a problem related to a common used library by both apps...

**Comment by jafd** [comment]
It’s the same on hosts with 13.5 beta, so my hopes that Apple will fix their framework are rather slim.

**Comment by jafd** [comment]
> There is also a "Device Support" update available from Apple Developer with the description "Install Device Support for macOS 14 beta 3 if installing the macOS Seed in a virtual Machine fails on a host Mac." that could be helpful here.

Nah, it's not helpful. I've ran into problems updating 12.6 to 12.6.7 with it installed. That bundle seems to do exactly what it says on the tin: be able to install macOS 14 beta.
